plugins/inventory/vmware_vm_inventory.py

```diff
@@ -136,6 +136,22 @@
                 - Also, transforms property name to snake case.
             type: bool
             default: False
+        proxy_host:
+          description:
+          - Address of a proxy that will receive all HTTPS requests and relay them.
+          - The format is a hostname or a IP.
+          - This feature depends on a version of pyvmomi>=v6.7.1.2018.12.
+          type: str
+          required: False
+          env:
+            - name: VMWARE_PROXY_HOST
+        proxy_port:
+          description:
+          - Port of the HTTP proxy that will receive all HTTPS requests and relay them.
+          type: int
+          required: False
+          env:
+            - name: VMWARE_PROXY_PORT
```

Comment:
@abikouo Would you please check and apply the same changes to `vmware_host_inventory` as well?